cmd/proofbundle/main.go

```diff
@@ -0,0 +1,204 @@
+// Copyright 2023 The Armored Witness authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// The proofbundle tool builds serialised proof bundles for use when
+// embedding the appled into the OS build, only useful for development work.
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/gob"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"os"
+
+	"github.com/transparency-dev/armored-witness-common/release/firmware"
+	"github.com/transparency-dev/armored-witness-common/release/firmware/ftlog"
+	"github.com/transparency-dev/merkle/rfc6962"
+	"github.com/transparency-dev/serverless-log/client"
+	"golang.org/x/mod/sumdb/note"
+	"k8s.io/klog"
+)
+
+var (
+	outputFile         = flag.String("output_file", "", "File to write the bundle to.")
+	logBaseURL         = flag.String("log_url", "", "Base URL for the firmware transparency log to use.")
```

Comment:
You may want to use `log_base_url` as it can be found on L126.

Reply:
Thanks for the catch, I did it the other way 'round and updated L126 :)